x/profile/client/cli/query.go

```diff
@@ -0,0 +1,74 @@
+package cli
+
+import (
+	"fmt"
+
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/client/context"
+	"github.com/cosmos/cosmos-sdk/client/flags"
+	"github.com/cosmos/cosmos-sdk/codec"
+	"github.com/desmos-labs/desmos/x/profile/internal/types"
+	"github.com/spf13/cobra"
+)
+
+// GetQueryCmd adds the query commands
+func GetQueryCmd(cdc *codec.Codec) *cobra.Command {
+	profileQueryCmd := &cobra.Command{
+		Use:                        types.ModuleName,
+		Short:                      "Querying commands for the profile module",
+		DisableFlagParsing:         true,
+		SuggestionsMinimumDistance: 2,
+		RunE:                       client.ValidateCmd,
+	}
+	profileQueryCmd.AddCommand(flags.GetCommands(
+		GetCmdQueryProfile(cdc),
+		GetCmdQueryProfiles(cdc),
+	)...)
+	return profileQueryCmd
+}
+
+// GetCmdQueryProfile queries a profile
+func GetCmdQueryProfile(cdc *codec.Codec) *cobra.Command {
+	return &cobra.Command{
+		Use:   "profile [address]",
```

Comment:
Can we rename this command to be 

```suggestion
		Use:   "[address]",
```

This way the user will be able to do `desmoscli query profile {address}` instead of `desmoscli query profile profile {address}`, making it more easy to execute. The documentation should be changed as well.